packages/position/src/js/__specs__/index.spec.js

```diff
@@ -26,7 +26,7 @@ describe('#below', () => {
     expect(below(targetEl).styleFor(el)).toEqual({
       position: 'absolute',
       top: '12px',
-      left: '-50px'
+      left: '0px'
```

Comment:
i was hoping these snapshots wouldn't change. maybe we need to setup the window width/height for these as well

Reply:
Actually the point was for these to change. There should never be time when the tooltip is offscreen.